holmes/config.py

```diff
@@ -53,6 +56,32 @@ class ConfigFile(RobustaBaseConfig):
     custom_runbooks: List[FilePath] = []
     custom_toolsets: List[FilePath] = []
 
+    @classmethod
+    def load_from_env(cls):
+        kwargs = {"llm": LLMType(os.getenv("HOLMES_LLM", "OPENAI").lower())}
+        for field_name, env_key in [
```

Comment:
keep it simple! why not just convert to uppercase for all the fields and do that?